tls/s2n_connection.h

```diff
@@ -148,6 +148,9 @@ struct s2n_connection {
      */
     uint16_t max_outgoing_fragment_length;
 
+    /* Negotiated TLS extension Maximum fragment size */
+    uint16_t max_fragment_length;
```

Comment:
I have removed max_fragment_length and replaced with uint8_t mfl_code to follow RFC6066